apps/developer-portal/src/constants/user-store-constants.ts

```diff
@@ -16,11 +16,37 @@
  * under the License.
  */
 
- /**
-  * The ID of the userstore type JDBC.
-  * 
-  * @constant
-  * 
-  * @type {string}
-  */
+/**
+ * The ID of the userstore type JDBC.
+ *
+ * @constant
+ *
+ * @type {string}
+ */
 export const JDBC = "JDBC";
+
+export const UserstoreTypeDescriptions = {
```

Comment:
`USERSTORE_TYPE_DESCRIPTIONS` do we have a plan to change all the constants uppercase ?

Reply:
As well as we need to localize these right?